### DISCUSSION TITLE
When was the last time The Australian Open Tennis was played on grass?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#322] [Galaxy Trivia] [answered]
**Hint:** almost four decades ago

**Comment by thuongtruong1009** [comment, marked as answer]
The last time the Australian Open Tennis was played on grass was in 1987. From 1988 onwards, the tournament has been played on hard courts, specifically the Plexicushion surface since 2008. The change from grass to hard courts was made to make the playing surface more consistent and durable, and to accommodate the increasing number of players who were primarily comfortable playing on hard courts.

**Comment by teafoot** [comment]
1987
